server/api/topics.js

```diff
@@ -59,6 +59,20 @@ router.get("/:topic_id/questions", (req, res) => {
 			res.json(500);
 		});
 });
+router.get("/:topic_id/questions/:question_id", (req, res) => {
+
+	const topicId = req.params.topic_id;
+	const questionId = req.params.question_id;
+	topicDb
+		.getQuestionById(topicId, questionId )
+		.then((data) => {
+			res.json(data);
+		})
+		.catch((err) => {
+			console.error(err);
+			res.json(500);
```

Comment:
This is probably forgotten :)

Reply:
I am working in topics part now cause it is in another branch

Reply:
Make sure that you are working on a specific branch solving a specific problem. After being merged start working on a new card/branch. This way you can avoid having a git branch hell that you have half changes here half in there ;)